.github/workflows/ci.yml

```diff
@@ -71,7 +71,7 @@ jobs:
 
     services:
       mysql:
-        image: mysql:8.0
+        image: mariadb:10.4
```

Comment:
What does this change?
Does it mean we should start using MariaDB everywhere instead of MySQL or is it for testing in CI only?